spec/lib/image_sizes_spec.rb

```diff
@@ -16,6 +16,7 @@
 
     before do
       described_class.class_variable_set(:@@cache, {})
+      allow(ENV).to receive(:[]).and_call_original
```

Comment:
It maybe worth adding the following after block:
``` Ruby
after do
  allow(ENV).to receive(:[]).and_call_original
end
```

This ensures that the stubb is cleared and subsequent tests that get any items from the ENV are not affected.

Reply:
RSpec mocks are per-example by default so no need to clear them up https://relishapp.com/rspec/rspec-mocks/v/3-6/docs/basics/scope